serverless.yml

```diff
@@ -25,29 +25,29 @@ plugins:
 provider:
   profile: ${env:AWS_PROVIDER_PROFILE, 'default'}
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs10.x
   apiGateway:
     # https://www.serverless.com/framework/docs/deprecations/#LOAD_VARIABLES_FROM_ENV_FILES
     shouldStartNameWithService: true
   stage: ${env:AWS_PROVIDER_STAGE, 'local'}
-  region: ${env:AWS_PROVIDER_REGION, 'local'}
+  region: ${env:AWS_PROVIDER_REGION, 'eu-west-1'}
   # To load environment variables externally
   # rename env.example to .env.local
   # docs: https://github.com/serverless/serverless/blob/master/docs/providers/aws/guide/variables.md
   # we use dotenv, docs: https://github.com/colynb/serverless-dotenv-plugin
   environment:
     SERVICE_VERSION: ${env:API_VERSION}
     AWS_PROVIDER_REGION: ${self:provider.region}
-    SERVICE: ${self:service}
+    SERVICE: ${env:SERVICE, 'trailers'}
```

Comment:
Can we delete this as it does not add much value in the service in fact. it was used in the early days to create the routing with express but will require some more devOps config that we (devs) do not control